R/get_carto.R

```diff
@@ -0,0 +1,553 @@
+#' Get a Carto basemap
+#'
+#' \code{get_carto} accesses a tile server for Carto basemaps and
+#' downloads/stitches map tiles/formats a map image.
+#'
+#' @param bbox a bounding box in the format c(lowerleftlon, lowerleftlat,
+#'   upperrightlon, upperrightlat).
+#' @param zoom a zoom level
+#' @param maptype light_all, dark_all, light_nolabels, light_only_labels, dark_nolabels, dark_only_labels,
+#' rastertiles/voyager, rastertiles/voyager_nolabels, rastertiles/voyager_only_labels, or rastertiles/voyager_labels_under.
+#' @param crop crop raw map tiles to specified bounding box. if FALSE, the
+#'   resulting map will more than cover the bounding box specified.
+#' @param messaging turn messaging on/off
+#' @param urlonly return url only
+#' @param color color or black-and-white (use force = TRUE if you've already
```

Comment:
specify type

Reply:
The function signature says this is an enum between `color` and `bw` so that should appear in the docstring.